guide/src/ecosystem/async-await.md

```diff
@@ -268,3 +269,191 @@ async fn main() -> pyo3::PyResult<()> {
     pyo3_asyncio::testing::main().await
 }
 ```
+
+
+# Calling async Python functions in Rust code
+
+In the previous chapter, we learnt how to call Python functions in Rust code. Now, we will look at two ways to call Python async functions from Rust code.
+In python, even an async function is treated like a regular function until called. When called, an async python function is treated as a coroutine.(More on this below.)
+We can either pass it either as a coroutine or either as a regular function.
+
+
+## Async Functions
+
+Before we start, let us take a look at the following snippet.
+
+```python
+async def fx():
+    return "hello, world"
+```
+
+Above, we have defined an async function. We will now execute it in rust code.
+
+In python, even an async function is treated like a regular function until called.
+e.g.
+
+```python
+>>> type(fx)
+<class 'function'>
+>>> type(fx())
+<class 'coroutine'>
+```
```

Comment:
The start of the `async / await` chapter already describes what an `async` Python is. I dont' think we need to repeat this, and can instead replace this with a summary / recap:

```suggestion
Above we learned that an async Python function is a special category of function which returns a `coroutine` object. We learned how to write these functions in Rust, so that they could be awaited in Python.
```

Your next sentence "we will now write Rust functions" is then a perfect continuation of the chapter.